### PR TITLE
Feature/tl 55

### PR DESCRIPTION
**Pull request by humulla**
Remove Equality related fields in package

### CODE REVIEW

File: Hackney.Shared.HousingSearch/Domain/Person/Identification.cs [hunk 1]
```diff
@@ -4,25 +4,25 @@ public class Identification
...
-        public static Identification Create(string identificationType, string value, bool originalDocumentSeen,
+        public static Identification Create(string identificationType, string value, bool isOriginalDocumentSeen,
             string linkToDocument)
         {
-            return new Identification(identificationType, value, originalDocumentSeen, linkToDocument);
+            return new Identification(identificationType, value, isOriginalDocumentSeen, linkToDocument);
         }
 
-        private Identification(string identificationType, string value, bool originalDocumentSeen, string linkToDocument)
+        private Identification(string identificationType, string value, bool isOriginalDocumentSeen, string linkToDocument)
```
Comment (amidomichael) [inline Hackney.Shared.HousingSearch/Domain/Person/Identification.cs]: I think `Identification` can be removed entirely (and also from swagger) as it's not used at all (it's not stored against the person https://app.swaggerhub.com/apis/Hackney/personAPI/1.0.0)

File: Hackney.Shared.HousingSearch/Domain/Person/Person.cs [hunk 2]
```diff
@@ -61,18 +53,8 @@ public class Person
 
         public string PreferredSurname { get; set; }
 
-        public decimal TotalBalance { get; set; }
```
Comment (amidomichael) [inline Hackney.Shared.HousingSearch/Domain/Person/Person.cs:64]: we need total balance as it's used by finance

File: Hackney.Shared.HousingSearch/Gateways/Models/Persons/QueryablePersonTenure.cs [hunk 3]
```diff
@@ -10,9 +10,6 @@ public class QueryablePersonTenure
         [Text(Name = "type")]
         public string Type { get; set; }
 
-        [Text(Name = "totalBalance")]
```
Comment (amidomichael) [inline Hackney.Shared.HousingSearch/Gateways/Models/Persons/QueryablePersonTenure.cs]: this should stay

**Review by LBHCallumM — APPROVED** [review]

File: Hackney.Shared.HousingSearch/Domain/Person/Person.cs [hunk 4]
```diff
@@ -34,14 +31,9 @@ public class Person
             Surname = surname;
             PreferredFirstname = preferredFirstname;
             PreferredSurname = preferredSurname;
-            TotalBalance = totalBalance;
```
Comment (amidomichael) [inline Hackney.Shared.HousingSearch/Domain/Person/Person.cs:37]: this should stay

File: Hackney.Shared.HousingSearch/Domain/Person/PersonTenure.cs [hunk 5]
```diff
@@ -25,8 +24,6 @@ private PersonTenure(string id, string type, decimal totalBalance, string startD
 
         public string Type { get; set; }
 
-        public decimal TotalBalance { get; set; }
```
Comment (amidomichael) [inline Hackney.Shared.HousingSearch/Domain/Person/PersonTenure.cs]: this should stay

File: Hackney.Shared.HousingSearch/Gateways/Models/Persons/QueryablePerson.cs [hunk 6]
```diff
@@ -41,24 +41,9 @@ public Person Create()
         [Text(Name = "preferredSurname")]
         public string PreferredSurname { get; set; }
 
-        [Text(Name = "totalBalance")]
-        public decimal TotalBalance { get; set; }
```
Comment (amidomichael) [inline Hackney.Shared.HousingSearch/Gateways/Models/Persons/QueryablePerson.cs:45]: this should stay

File: Hackney.Shared.HousingSearch/Domain/Person/Person.cs [hunk 7]
```diff
@@ -7,23 +7,20 @@ public class Person
         public Person() { }
 
         public static Person Create(string id, string title, string firstname, string middleName, string surname,
-            string preferredFirstname, string preferredSurname, decimal totalBalance, string ethinicity, string nationality,
```
Comment (amidomichael) [inline Hackney.Shared.HousingSearch/Domain/Person/Person.cs:10]: this should stay

File: Hackney.Shared.HousingSearch/Gateways/Models/Persons/QueryableIdentification.cs [hunk 8]
```diff
@@ -7,25 +7,25 @@ public QueryableIdentification()
 
         }
 
-        public static QueryableIdentification Create(string identificationType, string value, bool originalDocumentSeen,
```
Comment (amidomichael) [inline Hackney.Shared.HousingSearch/Gateways/Models/Persons/QueryableIdentification.cs:10]: what is this file, is it used?

Comment (humulla) [reply Hackney.Shared.HousingSearch/Gateways/Models/Persons/QueryableIdentification.cs:10]: It is not being used, I've removed it

**Review by amidomichael — APPROVED** [review]

**Review by mikemoody-amido — COMMENTED** [review]
Can we rename the `QueryableTenure` class in Hackney.Shared.HousingSearch.Gateways.Models.Assets to be `QueryableAssetTenure` to avoid name clashes when using both Assets and Tenure namespaces?

All good otherwise

**Pull request merged**